.circleci/config.yml

```diff
@@ -170,12 +172,19 @@ jobs:
             kubectl describe pods
       - run:
           when: on_fail
-          command: .circleci/cleanup-<< parameters.cloud_provider >>.sh
-
+          command: |
+            if [ "x<< parameters.cloud_provider >>" == "xgke" ]; then
+              .circleci/cleanup-gke.sh "$GCLOUD_SERVICE_KEY" "${CIRCLE_SHA1:8:8}" "${GOOGLE_PROJECT_ID}" "${GOOGLE_COMPUTE_ZONE}" "dev-testing-${CIRCLE_SHA1:8:8}-<< parameters.cluster_id >>"
+            else
+              .circleci/cleanup-<< parameters.cloud_provider >>.sh
+            fi
     environment:
       PACKET_CLUSTER_ID: "<< parameters.cluster_id >>"
       KUBECONFIG: /home/circleci/project/data/kubeconfig
       GO111MODULE: "on"
+      GOOGLE_COMPUTE_ZONE: "europe-west3-a"
```

Comment:
Can you make GKE specifics parameters with default values too?

Reply:
So you can do smething like:
```yaml
  integration-tests:
    parameters:
      cloud_provider:
        type: string
      cluster_id:
        type: string
      test_tags:
        type: string
      gke_compute_zone:
        type: string
        default: "europe-west3-a"
      gke_project_id:
        type: string
        default: "ci-management"
....
    environment:
...
	      GOOGLE_COMPUTE_ZONE:  "<< parameters. gke_compute_zone >>"
             GOOGLE_PROJECT_ID: "<< parameters. gke_project_id >>"
```

Reply:
Thanks, will do.